lib/GLPI/Agent/SNMP/MibSupport/Ubnt.pm

```diff
@@ -32,7 +58,56 @@ sub getSerial {
 sub getMacAddress {
     my ($self) = @_;
 
-    return $self->get(ubntWlStatApMac);
+    return getCanonicalMacAddress($self->get(ubntWlStatApMac));
+}
+
+sub run {
+    my ($self) = @_;
+
+    my $device = $self->device
+        or return;
+
+    # Get list of device ports (e.g. raX, raiX etc.)
+    my $ports = $device->{PORTS}->{PORT};
+
+    # Get list of SSID
+    my $unifiVapEssidValues = $self->walk(unifiVapEssid) || {};
+    # Get list of Radios (e.g. ra0, rai0 etc.)
+    my $unifiVapNameValues = $self->walk(unifiVapName) || {};
+    # The list of Radios is co-related to the list of SSIDs
+    # $unifiVapNameValues->{0} = ra0
+    # $unifiVapEssidValues->{0} = <SSID>
+
+    foreach my $port (keys(%$ports)) {
+        # For each device Radio port (raX, raiX etc.)
+        # If you have more than one SSID there will also be more raX, raiX for each SSID.
+        my $ifdescr = $device->{PORTS}->{PORT}->{$port}->{IFDESCR};
+        next unless defined($ifdescr) && $ifdescr =~ /^ra/;
+
+        # Replaces the port iftype from "Ethernet" (6) to "WiFi" (71)
+        if ($device->{PORTS}->{PORT}->{$port}->{IFTYPE} == 6) {
+            $device->{PORTS}->{PORT}->{$port}->{IFTYPE} = 71
+        }
```

Comment:
Just in case IFTYPE is not defined, and you missed a semicolon on line 89:
```suggestion
        if ($device->{PORTS}->{PORT}->{$port}->{IFTYPE} && $device->{PORTS}->{PORT}->{$port}->{IFTYPE} == 6) {
            $device->{PORTS}->{PORT}->{$port}->{IFTYPE} = 71;
        }
```